packages/web/src/components/result/ReactiveList.js

```diff
@@ -212,7 +212,9 @@ class ReactiveList extends Component {
 
 					if (nextProps.hits.length < this.props.hits.length) {
```

Comment:
I think @divyanshu013 and I discussed this in the related issue to change this `if` condition to have shallow equal comparison between the two `hits`.